spec/models/redis_store_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe RedisStore do
+  let(:klass) do
+    Class.new(RedisStore) do
+      redis_store 'my_namespace'
+      default_ttl 60
+      attribute :uuid
+      attribute :email
+      alias redis_key uuid
+    end
+  end
+
+  subject { klass.new(uuid: 'e66fd7b7-94e0-4748-8063-283f55efb0ea', email: 'foo@bar.com') }
+
```

Comment:
I think we should raise exceptions when an instance is created and the class level accessors are not set. Some specs for that would be nice as well. See Common::Exceptions::BaseError for a description of what I mean.